test-requirements.txt

```diff
@@ -1,3 +1,3 @@
-coverage>=4.0,<4.4
+coverage>=6.0,<6.4
 mock
 pytest-cov<2.6.0
```

Comment:
I think we also need to unpin `pytest-cov`, I remember those specific versions of `pytest-cov` and `coverage` worked together but I assume we can now unpin both of them, their latest versions work together.

Reply:
Ok, done that as well. Do you know why the coverage pipelines are now pending? :)